src/main/java/com/example/entity/Product.java

```diff
@@ -14,15 +13,29 @@ public class Product {
     private String productName;
     private Double productPrice;
 
+    @ManyToOne
+    @JoinColumn(name = "category_id")
```

Comment:
While giving bidirectional mapping, think about it really required bidirectional mapping